include/sparrow/layout/array_base.hpp

```diff
@@ -127,6 +128,9 @@ namespace sparrow
         const_bitmap_range bitmap() const;
         const_value_range values() const;
 
+        [[nodiscard]] std::optional<const std::string_view> name() const;
+        [[nodiscard]] std::optional<const std::string_view> metadata() const;
```

Comment:
I think `std::optional<std:string_view>` would be enough since `std::string_view` only provides a const API.

Reply:
done